Assets/Resources/Scripts/SceneController.cs

```diff
@@ -0,0 +1,67 @@
+﻿using UnityEditor;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class SceneController : MonoBehaviour
+{
+	// currentScene populates in Start, previous and next need to be set manually
```

Comment:
the SceneController should probably be the one responsible for deciding which scene to go to next at any given step, for starters an array that the SceneController (which would be on a DontDestroyOnLoad object) steps forward and back through